uportal-war/src/main/java/org/apereo/portal/portlets/backgroundpreference/BackgroundSetSelectionStrategy.java

```diff
@@ -33,6 +33,9 @@
 
     String[] getImageThumbnailSet(PortletRequest req);
 
+    @since 4.3.1
```

Comment:
The many of the remaining ADA tasks are targeting release `4.3.2`, because they are considered fixes instead of new features.